drools-yaml-rules-runtime/src/main/java/org/drools/yaml/runtime/jpy/AstRulesEngine.java

```diff
@@ -12,17 +7,15 @@
 import java.util.Map;
 import java.util.function.Function;
 
-import static org.drools.yaml.core.RulesExecutor.OBJECT_MAPPER;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.drools.yaml.api.RulesExecutor;
+import org.drools.yaml.api.RulesExecutorContainer;
+import org.kie.api.runtime.rule.Match;
 
 public class AstRulesEngine {
 
     private Iterator<Map<String, Map>> lastResponse = Collections.emptyIterator();
 
-    public long createRuleset(String rulesetString) {
```

Comment:
I think we need to keep this around before because it is used on the Python end. Better to `@Deprecate` and _then_ remove it after an iteration.